src/Compilers/VisualBasic/Portable/Symbols/PropertySymbol.vb

```diff
@@ -409,6 +409,26 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Symbols
         End Property
 
         ''' <summary>
+        ''' Is this a property of a tuple type?
+        ''' </summary>
+        Public Overridable ReadOnly Property IsTupleProperty() As Boolean
+            Get
+                Return False
+            End Get
+        End Property
```

Comment:
fixed

---
In reply to: [71416035](https://github.com/dotnet/roslyn/pull/12596#discussion_r71416035) [](ancestors = 71416035)